3.6.1/Dockerfile

```diff
@@ -0,0 +1,137 @@
+#
+# Dockerfile for Python
+#
+FROM ubuntu:16.04
```

Comment:
It seems like a lot of work could be saved and a more standardized result achieved by inheriting from library/python or something similar. I know that's not always the case, but want to make sure that that route was explored.